MotorControle/OpenCr_MotorControle/main.ino

```diff
@@ -27,6 +14,27 @@ SerialComm SC(&Serial);
 #define STRING_BUF_NUM 64
 String cmd[STRING_BUF_NUM];
 
+#include "SerialComm.h"
+#include "ControlMessage.h"
+#include "MessageIO.h"
+
+#define BAUD 57600 
+
+MessageIO IO;
+SerialComm SC(&Serial);
+
+// Definition des defines pour la communication
+// Definition of define for communication
+#define SETLIMITESINDEX 1
```

Comment:
tu pourrait mettre des _ entre chaque mot pour que ça soit plus lisible genre RETURN_HOME_POSITION_INDEX

Reply:
utilise find-and replace pour les changé